staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,10 +94,14 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.StringSliceVar(&a.EnablePlugins, "enable-admission-plugins", a.EnablePlugins, ""+
 		"admission plugins that should be enabled in addition to default enabled ones. "+
+		"Default enabled plugins: NamespaceLifecycle, LimitRanger, ServiceAccount, DefaultStorageClass, PersistentVolumeClaimResize, "+
```

Comment:
Please derive this list programmatically. Compare pkg/kubeapiserver/options/plugins.go.

Reply:
Agree with @sttts

Reply:
@dixudx I try to import the `pkg/kubeapiserver/options` but it will meet import cycle error, how should i do?

Reply:
> @dixudx I try to import the pkg/kubeapiserver/options but it will meet import cycle error, how should i do?

importing internal pkgs from staging area is not suggested :-)

Reply:
@yue9944882 yes, that's what i mean. I don't know how to get the list there.(add a link to flag description which link to that list? It seems bad too.) Any recommend solution?

Reply:
can you call `a.enabledPluginNames()`?